playground/mocks/data/idp/idx/error-authenticator-webauthn-failure.json

```diff
@@ -175,7 +175,7 @@
       "type":"security_key",
       "key":"webauthn",
       "id":"aut12uqz8PnlDpzQ20g4",
-      "displayName":"MacBook Touch ID",
+      "displayName":"Security Key or Biometric Authenticator",
```

Comment:
I think you added "Authenticator" by mistake to this one.

Reply:
Thank you!